src/driver.ml

```diff
@@ -547,43 +547,6 @@ let map_signature sg =
    | Entry points                                                    |
    +-----------------------------------------------------------------+ *)
 
-let mapper =
-  let module Js = Ppxlib_ast.Selected_ast in
-  (*$*)
-  let structure _ st =
-    Js.of_ocaml Structure st
-    |> map_structure
-    |> Js.to_ocaml Structure
-  in
-  (*$ str_to_sig _last_text_block *)
-  let signature _ sg =
-    Js.of_ocaml Signature sg
-    |> map_signature
-    |> Js.to_ocaml Signature
-  in
-  (*$*)
-  { Ocaml_common.Ast_mapper.default_mapper with structure; signature }
-;;
-
-let as_ppx_rewriter_main argv =
-  let argv = Caml.Sys.executable_name :: argv in
-  let usage =
-    Printf.sprintf "%s [extra_args] <infile> <outfile>" exe_name
-  in
-  match
-    Arg.parse_argv (Array.of_list argv) (Arg.align (List.rev !args))
```

Comment:
Why do we reverse the list of arguments here? I've preserved that, but I don't know why that's done.

Reply:
Because the list is in reverse order as the order in which arguments are registered; `add_arg` adds the argument at the beginning of the list.

We could make this more explicit by hidding the `args` reference and only exposing a function that return the list of arguments in the right order. Then the `List.rev` would be closer to the `add_args` function, making the code clearer.

Reply:
Oh right, thanks! I guess that kind of function is already implemented and called `get_args` but not used here, right? Would it make sense to use it here the same way as done in `standalone_run_as_ppx_rewriter`?

Reply:
I guess `get_args` really only makes sense for the standalones. I've left that as it was.